index.js

```diff
@@ -36,10 +37,9 @@ setInterval(async function (){
     let locationTable = await db.getLocations()
     let locationsArray = []
     for(let i = 0; i < locationTable.length; i++){
-        locationsArray.push(locationTable[i].locationid)
+        locationsArray.push(locationTable[i].locationId)
```

Comment:
I think that the "i" in `locationid` should be small.

Reply:
Oh yeah, I initially had `locationId` when I created the Location class but then changed it for consistency because there is no underscore in the db column name